content/about/working-groups.md

```diff
@@ -0,0 +1,245 @@
+---
+title: Working Groups
+description: "Working Groups| Node.js"
+authors: williamkapke,  Trott,  fhemberger,  rxmarbles, mhdawson,XhmikosR,ryanmurakami,outsideris,MaledongGit, vsemozhetbyt, wonderdogone,sotayamashita,richardlau, pierreneter,nschonni, marocchino, stevemao, lpinca, phillipj, jasnell,sejaljain123
```

Comment:
nit: spacing between the author names.

Reply:
Getting this false positive error!
![image](https://user-images.githubusercontent.com/56127301/119094647-eea53b00-ba2e-11eb-92a0-508b3bddd9aa.png)
![image](https://user-images.githubusercontent.com/56127301/119094804-1e544300-ba2f-11eb-8bee-ff3ada4b5e34.png)

If this is not allowed, can I add MD file to remarkignore?

Reply:
@Trott or @nschonni  What is the proper way to fix this linting error? 
is it with backticks? `nodejs/ci-config-travis`

Reply:
Yeah, I think code sections makes the most sense or just flip them to URLs to the repos